src/modules/Dropdown/Dropdown.js

```diff
@@ -187,14 +187,15 @@ export default class Dropdown extends Component {
     if (process.env.NODE_ENV !== 'production') {
       // in development, validate value type matches dropdown type
       const isNextValueArray = Array.isArray(nextProps.value)
+      const hasValue = _.has(nextProps, 'value')
 
       /* eslint-disable no-console */
-      if (nextProps.multiple && !isNextValueArray) {
+      if (hasValue && nextProps.multiple && !isNextValueArray) {
```

Comment:
We should not throw these dev errors regarding value unless there is actually a value.